addons/l10n_ar_withholding/wizards/account_payment_register.py

```diff
@@ -89,3 +93,24 @@ def _get_conversion_rate(self):
                 self.payment_date,
             )
         return 1.0
+
+    @api.depends('partner_id', 'payment_date')
+    def _compute_l10n_ar_withholding_ids(self):
+        """ Primero estoy haciendo la lógica para retenciones de ganancias aplicadas en pagos de proveedor. """
+        # Busco fecha de pago y en base a eso vemos si hubo retenciones en el mes
+        date = fields.Date.from_string(self.payment_date) or datetime.date.today()
+        # Vemos los impuestos que tiene el partner en la vista form, sección accounting de res.partner en tablita Withholding (AR)
+        is_vendor_payment = False
+        move_lines_to_pay_ids = self._context['active_ids']
```

Comment:
We can not use line_ids.move_id ?

Reply:
Yes, thanks for the suggestion @jco-odoo .